Cargo.toml

```diff
@@ -27,7 +27,7 @@ path = "src/lib.rs"
 crate-type = ["rlib"]
 
 [dependencies]
-serde = { version = "1.0.123", features = ["derive"] }
+serde = { version = "1.0.123", features = ["derive"], optional = true }
```

Comment:
This is a breaking change.

We could make it non-breaking by enabling the `serde` feature by default, but that would also enable `small-vec/serde` and `cssparser/serde`, which are not currently enabled by default.

It looks like `lightningcss` is on an alpha train. 
Is a small breaking change here acceptable?

Unless `serde` is commonly used for some reason, I think it makes sense to have it be disabled by default.